From e88973ae85d8a160473bb71ad56385d27ff4f911 Mon Sep 17 00:00:00 2001
From: tpaviot <tpaviot@gmail.com>
Date: Thu, 12 Jan 2012 22:01:25 +0100
Subject: [PATCH] [cppcheck-error-fix][instance-destroyed-immediately]

[Poly/Poly_CoherentTriangulation.cxx:578]: (error) instance of "Standard_ProgramError" object destroyed immediately
[Poly/Poly_CoherentTriangulation.cxx:653]: (error) instance of "Standard_ProgramError" object destroyed immediately
---
 src/Poly/Poly_CoherentTriangulation.cxx |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Poly/Poly_CoherentTriangulation.cxx b/src/Poly/Poly_CoherentTriangulation.cxx
index 777d497..9c442ef 100644
--- a/src/Poly/Poly_CoherentTriangulation.cxx
+++ b/src/Poly/Poly_CoherentTriangulation.cxx
@@ -575,7 +575,7 @@ void Poly_CoherentTriangulation::RemoveLink (Poly_CoherentLink& theLink)
         else if (iNode == pTri[i]->Node(2))
           const_cast<Poly_CoherentTriangle *>(pTri[i])->mypLink[2] = 0L;
         else
-          Standard_ProgramError("Poly_CoherentTriangulation::RemoveLink: "
+          Standard_ProgramError::Raise("Poly_CoherentTriangulation::RemoveLink: "
                                 " wrong connectivity between triangles");
       }
     }
@@ -650,7 +650,7 @@ Standard_Boolean Poly_CoherentTriangulation::FindTriangle
         else if (aTri.Node(1) == theLink.Node(1))
           pTri[1] = &aTri;
       } else
-        Standard_ProgramError("Poly_CoherentTriangulation::FindTriangle : "
+        Standard_ProgramError::Raise("Poly_CoherentTriangulation::FindTriangle : "
                               " Data incoherence detected");
       if (pTri[0] && pTri[1])
         break;
-- 
1.7.9.1

